tests/waku_core/test_time.nim

```diff
@@ -21,14 +21,10 @@ suite "Waku Core - Time":
 
     ## When
     let
-      timeInSecondsInt = secondsPart.int
       timeInSecondsInt64 = secondsPart.int64
-      timeInSecondsFloat = float(secondsFloat)
       timeInSecondsFloat64 = float64(secondsFloat)
 
     ## Then
     check:
-      getNanosecondTime(timeInSecondsInt) == lowResTimestamp
```

Comment:
Why this check is not needed?

Reply:
Removed these tests, as they are in contradiction with the change ( allow using the function on 32bit )